src/inmanta/agent/agent.py

```diff
@@ -79,11 +81,13 @@ def __str__(self) -> str:
 class ResourceActionBase(abc.ABC):
     """Base class for Local and Remote resource actions"""
 
+    # Model dependencies between resources + inform each other when they're done deploying
+
     resource_id: Id
     future: ResourceActionResultFuture
     dependencies: list["ResourceActionBase"]
     # resourceid -> attribute -> {current: , desired:}
-    changes: dict[ResourceVersionIdStr, dict[str, AttributeStateChange]]
+    # changes: dict[ResourceVersionIdStr, dict[str, AttributeStateChange]]
```

Comment:
These commented lines have to be cleaned up

Reply:
We have a discussion over [there](https://github.com/inmanta/inmanta-core/pull/7101#discussion_r1494916852) about these fields. They aren't used anymore are they ?